core/utils/toolbox.ts

```diff
@@ -24,8 +24,6 @@ export interface BlockInfo {
   disabledReasons?: string[];
   enabled?: boolean;
   id?: string;
-  x?: number;
-  y?: number;
```

Comment:
What does member removal mean for backward compatibility for users upgrading to v12? I feel that I don't have a good enough context to know this already.

Reply:
Makes sense--thanks for clarfiying!